### PR TITLE
[SQL] Implement "DESCRIBE MODEL" command

### PR DESCRIPTION
**Pull request by eddyxu**

### CODE REVIEW

**Review by chunyang — APPROVED** [review]
👍

File: src/main/scala/ai/eto/rikai/sql/spark/execution/ModelCommand.scala [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+      session.conf.get(
+        Catalog.SQL_ML_CATALOG_IMPL_KEY,
+        Catalog.SQL_ML_CATALOG_IMPL_DEFAULT
+      )
+    )
+}
+
+object ModelCommand {
+
+  // Shared between "DESCRIBE MODEL" and "SHOW MODELS"
```
Comment (bobingm) [inline src/main/scala/ai/eto/rikai/sql/spark/execution/ModelCommand.scala:38]: noted

File: src/main/scala/ai/eto/rikai/sql/spark/execution/DescribeModelCommand.scala [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+
+case class DescribeModelCommand(name: String) extends ModelCommand {
+
+  override val output: Seq[Attribute] = ModelCommand.output
+
+  override def run(session: SparkSession): Seq[Row] = {
+    catalog(session).getModel(name) match {
+      case Some(model) => Seq(Row(model.name, model.uri))
+      case None =>
+        throw new ModelNotFoundException(s"Model '${name}' not found")
```
Comment (bobingm) [inline src/main/scala/ai/eto/rikai/sql/spark/execution/DescribeModelCommand.scala:31]: What will user see if we raise exception here?

Comment (eddyxu) [reply src/main/scala/ai/eto/rikai/sql/spark/execution/DescribeModelCommand.scala:31]: you can try `spark.sql("DESCRIBE TABLE no_exist")`, it throws an `AnalyzeException` exception as well.

**Pull request merged**